x-pack/plugins/lists/public/exceptions/components/builder/entry_renderer.tsx

```diff
@@ -338,13 +339,16 @@ export const BuilderEntryItem: React.FC<EntryItemProps> = ({
         );
       case OperatorTypeEnum.WILDCARD:
         const wildcardValue = typeof entry.value === 'string' ? entry.value : undefined;
-        let os: OperatingSystem = OperatingSystem.WINDOWS;
-        if (osTypes) {
-          [os] = osTypes as OperatingSystem[];
+        let actualWarning: React.ReactNode | string | undefined;
```

Comment:
I know you didn't introduce wildcard here, but can we add a unit test for it in the existing `entry_renderer` tests?

Reply:
done!